esp/logging/loggingagent/espserverloggingagent/loggingagent.hpp

```diff
@@ -54,7 +54,12 @@ class CTransIDBuilder : public CInterface, implements IInterface
         if (value)
             id.append(value->get()).append('-');
         else
-            id.append('??-');
+        {
+            const char* ptr = key;
+            if (strlen(key) > 11) //skip 'Transaction'
```

Comment:
we might want to expand upon the comment... skiping the "transaction" prefix of the key